ENIGMAsystem/SHELL/Universal_System/bufferstruct.cpp

```diff
@@ -256,6 +267,12 @@ void buffer_get_surface(int buffer, int surface, int mode, unsigned offset, int
   //TODO: Write this function
 }
 
+//NOTE: This function should most likely be added in graphics systems.
+void buffer_set_surface(int buffer, int surface, int mode, unsigned offset, int modulo) {
+  //get_buffer(binbuff, buffer);
```

Comment:
If you're going to put a stub it's a good habit to put DEBUG_MESSAGE("Unimplemented", M_WARNING) or whatever in it to lessen confusion when people's games call it and it doesn't work.

Reply:
on it as we speak, thanks!